pom.xml

```diff
@@ -126,6 +126,12 @@
       <artifactId>jackson-datatype-jsr310</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.godaddy</groupId>
```

Comment:
Worth waiting for https://github.com/ONSdigital/rm-common-config/pull/8?

Reply:
Done 👍